src/TodoContainer.js

```diff
@@ -0,0 +1,99 @@
+/* eslint-disable no-param-reassign */
+import { useState } from 'react';
+import { v4 as uuidv4 } from 'uuid';
+import { Routes, Route } from 'react-router-dom';
+import TodoList from './components/TodoList';
+import Header from './components/Header';
+import InputTodo from './components/InputTodo';
+import About from './functionBased/About';
+import NotMatch from './functionBased/NotMatch';
+import Navbar from './components/Navbar';
+
+let todoArray = JSON.parse(localStorage.getItem('todoArray') || '[]');
+const componentDidMount = () => {
```

Comment:
Please change the name of the function to something that corresponds to its purpose more. `componentDidMount` function is a lifecycle method of component classes. You don't want to use this name.